src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Defaults.cs

```diff
@@ -86,7 +86,6 @@ private static JsonSerializerOptions CreateDefaultOptions()
     [JsonSerializable(typeof(ChatResponseUpdate))]
     [JsonSerializable(typeof(IReadOnlyList<ChatResponseUpdate>))]
     [JsonSerializable(typeof(Dictionary<string, object>))]
-    [JsonSerializable(typeof(IDictionary<int, int>))]
```

Comment:
Are any of the others unnecessary?

Reply:
It's possible, although hard to tell at a glance.